Bottom: 8303c127907d0f81aa6be6ddaf240dafb8414d8e
Top:    02611204597d30f04886d63f142d6258db2e330f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-14 18:41:05 +0000

Refresh of add-flag-parameters-to-resetfo

---

diff --git a/src/toporouter.c b/src/toporouter.c
index 0a83569..7f2081b 100644
--- a/src/toporouter.c
+++ b/src/toporouter.c
@@ -2543,7 +2543,7 @@ void
 import_clusters(toporouter_t *r)
 {
   NetListListType nets;
-  ResetConnections (false);
+  ResetConnections (false, DRCFLAG); /* XXX: NOT SET? */
   nets = CollectSubnets(false);
   NETLIST_LOOP(&nets);
   {
